### ISSUE TITLE
Add `time-min-milliseconds` option to "whitelist" values.

### ISSUE THREAD

**Issue by stormwarning** [stylelint/stylelint#2327]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

`time-min-milliseconds` is a great rule to enforce best-practices, but it would be helpful to have the option to allow specific values that are lower than the set minimum—specifically, zero, although there may be cases where other values might be needed.

My particular use-case is when choreographing multiple elements/props transitions on state changes.

> Which rule, if any, is this issue related to?

`time-min-milliseconds`

> What CSS is needed to reproduce this issue?

```css
.foo {
  transition-delay: 0ms;
}
```

> What stylelint configuration is needed to reproduce this issue?

Perhaps something like:
```json
{
  "rules": {
    "time-min-milliseconds": [
      100, 
      {
        "except": [ 0 ]
      }
    ]
  }
}
```

> Which version of stylelint are you using?

`7.8.0`

**Comment by hudochenkov** [comment]
Thank you for the report and for using the template.

With config like this:

```json
{
  "rules": {
    "time-min-milliseconds": 100
  }
}
```

And stylesheet:

```css
.foo {
  transition-delay: 0ms;
  animation-delay: 10ms;
}
```

You will receive warning only for `animation-delay`.

You may disable this rule for specific lines, like so:

```css
.foo {
  transition-delay: 0ms;
  animation-delay: 10ms; /* stylelint-disable-line time-min-milliseconds */
}
```

**Comment by evilebottnawi** [comment]
@hudochenkov `transition-delay: 0ms;` used for disable delay (run animation immediately), i think best here add option for this.
@stormwarning btw you can use `transition-delay: initial;`

**Comment by stormwarning** [comment]
@hudochenkov 🤦‍♂️ You're correct, `time-min-milliseconds` doesn't throw a warning for a 0ms value. Looks like my config still had the deprecated `time-no-imperceptible` rule in it though...

Thanks for the confirmation!
